### DISCUSSION TITLE
Unit of density in PeleC

### DISCUSSION THREAD

**Discussion by jagmohas** [AMReX-Combustion/PeleC#608] [Q&A] [answered]
I am trying EB-C3 tutorial which uses Fuego Eos model. To make it easier to understand, I changed the chemistry model to air and changed Y_init_O2 and Y_init_N2 to that of air. The simulation is initialized with P=1013250.0 and T=1500 for which I should expect density to be ~2.35 kg/m^3, but I get 0.00023370754 in PeleC (PeleC reports min, max and also verified in plt). What is the unit of   density used by PeleC? Sounds speed also comes way higher (75382.84375). Do I need to specify a scaling somewhere?

**Comment by baperry2** [comment, marked as answer]
PeleC uses CGS units, so the density is in units of g/cm3 (also the pressure you are specifying is equivalent to 1 atm, and the sound speed is 75382 cm/s, or 753 m/s, which makes sense for those conditions).
